pages/implementation/devs-101.mdx

```diff
@@ -0,0 +1,286 @@
+---
+layout: 'guide'
+title: 'Avo 101 for developers'
+# abstract: 'Developer's overview of Avo'
+---
+
+import Link from '../../components/Link';
+
+The use case of Avo for developers is twofold.
+
+- Avo generated file, which is the code representation of your tracking plan, containing Avo Functions
+- Avo Inspector SDKs
+
+Each can be used on its own, but together they unlock the full power of Avo.
+
+## <a name="avo-generated-file"></a> Avo generated file containing Avo Functions
+
+This is the code representation of the tracking plan you define in Avo for a given source. You can think of a source as a platform that sends events. For example if you set up an iOS/Swift source the generated file would be a `.swift` file. Since the tracking plans are different for each customer the content of this file is also different. We generate the files and you can download them either with <Link passHref href="/implementation/cli"><a>the Avo CLI</a></Link> or in the Avo web interface in <Link passHref href="/workspace/implement"><a>the Implement tab</a></Link>.
+
+First of all, feel free to explore your generated file. It's deliberately designed to be easily readable. Our code generation is under active development and some advanced or new features may slightly differ from one platform to another. The generated file is the best source of truth when looking for implementation details!
+
+You can make changes to the generated file, but remember, every time you pull the code from Фvo you would need to apply your changes separately. If you find yourself in this situation please <Link passHref href="/help/troubleshooting"><a>contact us</a></Link> and we will make our best to make the generated code support your use case.
+
+### Avo Command Line Interface
+
+We created a CLI tool to interact with Avo from the command line.
+Run
+
+```sh
+$ npm install -g avo
+```
+
+to install the CLI.
+Then login with
+
+```sh
+$ avo login
+```
+
+And pull the Avo Generated file with the following command
+
+```sh
+$ avo pull
+```
+
+it will guide you through initial the setup and will create a config file called `avo.json` in the directory. Next time when you pull the setup will be picked from that file.
```

Comment:
```suggestion
it will guide you through initial the setup and will create a config file called `avo.json` in the directory. Next time when you pull the setup will be picked from that file. 

Make sure to check the `avo.json` file in to your source control to make sure the Avo configuration is in sync across your team.
```